src/import/k8s.ts

```diff
@@ -163,5 +163,9 @@ const X_GROUP_VERSION_KIND = 'x-kubernetes-group-version-kind';
 async function downloadSchema(apiVersion: string) {
   const url = `https://raw.githubusercontent.com/awslabs/cdk8s/master/kubernetes-schemas/v${apiVersion}/_definitions.json`;
   const output = await download(url);
-  return JSON.parse(output) as JSONSchema4;
+  try {
+    return safeParseJsonSchema(output) as JSONSchema4;
```

Comment:
Why not change the return type of `safeParseJsonSchema` to JSONSchema4? Or is that not technically type safe

Reply:
Should be fine. Changed 👍